tests/integration/kubernetes/gha-run.sh

```diff
@@ -86,6 +86,8 @@ function deploy_kata() {
     yq write -i "${tools_dir}/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml" 'spec.template.spec.containers[0].env[4].value' --tag '!!str' "true"
     # Let the `kata-deploy` create the default `kata` runtime class
     yq write -i "${tools_dir}/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml" 'spec.template.spec.containers[0].env[5].value' --tag '!!str' "true"
+    # Ensure USING_NFD is set according to the GHA env var
+    yq write -i "${tools_dir}/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml" 'spec.template.spec.containers[0].env[6].value' --tag '!!str' "${USING_NFD}"
```

Comment:
You will help me immensely  in https://github.com/kata-containers/kata-containers/pull/7621 if you set a default value to `USING_NFD`.